### PR TITLE
tests for gutters in grid layout

### PR DESCRIPTION
**Pull request by rachelandrew**
I wanted to add tests for issues [6](https://github.com/rachelandrew/gridbugs#6-the-grid-gap-property-should-accept-percentage-values) and [7](https://github.com/rachelandrew/gridbugs#7-grid-gaps-should-behave-as-auto-sized-tracks) on GridBugs. I couldn't find any gutter tests so made some.

With the CSS WG resolving to change `grid-gap` to `gap`, `grid-row-gap` to `row-gap` and `grid-column-gap` to `column-gap` it seemed sensible to have duplicates of each test testing the required aliases for the gutter implementation in Grid layout.

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/18598)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/276275887)

**Comment by tomalec** [comment]
Wouldn't it be safer to use `display: block` in the references to make those tests more independent from [tests](https://github.com/w3c/web-platform-tests/tree/master/css/css-grid/abspos) for ["9. Absolute Positioning"](https://www.w3.org/TR/css-grid-1/#abspos)?

**Comment by rachelandrew** [comment]
I'm not sure I understand?

**Comment by tomalec** [comment]
In `reference/grid-*-gutters-ref.html` you use `display: grid`
https://github.com/rachelandrew/web-platform-tests/blob/38fae50f0f91345961d091cd10cd86ee8cf06325/css/css-grid-1/reference/grid-collapsed-row-gutters-ref.html
```css
    #grid {
        display: grid;
        ...
        position: relative;
    }
```

I suggest removing `display: grid`
```css
    #grid {
        /*display: block;*/
        ...
        position: relative;
    }
```


So, for example, if `gap` works correctly, but the absolute position in the grid does not, this test will pass. 
(While some tests from https://github.com/rachelandrew/web-platform-tests/tree/38fae50f0f91345961d091cd10cd86ee8cf06325/css/css-grid-1/abspos will fail)

**Comment by rachelandrew** [comment]
I think that was essentially just a C&P error which I'd then managed to copy across to the three based on the same example. I've removed it and they render correctly. Thanks!

**Review by tomalec — APPROVED** [review]
LGTM, if my vote (as not-participant) counts

**Comment by gsnedders** [comment]
@rachelandrew can you rebase this so the test is moved to `css/css-grid` as with the rest of the testsuite now?

**Comment by Ms2ger** [comment]
@rachelandrew it looks like something went wrong here git-wise. Can you get rid of the unrelated commits? Let us know if you'd like help with that.

**Comment by rachelandrew** [comment]
I probably do need some help as I have no idea what happened there.

**Comment by Ms2ger** [comment]
I pushed a rebase to <https://github.com/w3c/web-platform-tests/compare/gridbugs>; you could push that to your fork yourself, or [allow us to do it](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/).

**Comment by rachelandrew** [comment]
I've allowed edits from maintainers - thanks

**Review by Ms2ger — APPROVED** [review]
I'm going to assume @tomalec's opinion counts.

**Pull request merged**
